tests/std/tests/P0896R4_ranges_alg_find_end/test.cpp

```diff
@@ -71,9 +72,29 @@ constexpr void smoke_test() {
     }
 }
 
+constexpr void test_1559808() {
+    // Regression test for DevCom-1559808, an interaction between vector and the
+    // use of structured bindings in the constexpr evaluator.
+
+    std::vector<int> haystack(33, 42), needle(8, 42); // No particular significance to any numbers in this function

```

Comment:
We conventionally avoid defining multiple variables on a single line.

Also occurs in `P0896R4_ranges_alg_search/test.cpp`.